tests/integration/attendanceSheets.test.js

```diff
@@ -492,6 +512,27 @@ describe('ATTENDANCE SHEETS ROUTES - POST /attendancesheets', () => {
         course: coursesList[2]._id.toHexString(),
         file: 'test',
         date: '2020-01-25T09:00:00.000Z',
+        trainer: userList[3]._id.toHexString(),
+      };
+
+      const form = generateFormData(formData);
+
+      const response = await app.inject({
+        method: 'POST',
+        url: '/attendancesheets',
+        payload: getStream(form),
+        headers: { ...form.getHeaders(), Cookie: `alenvi_token=${authToken}` },
+      });
+
+      expect(response.statusCode).toBe(403);
+    });
+
+    it('should return a 403 if trainer in payload is not course trainer', async () => {
```

Comment:
Quelle est la diff avec le test : should return a 403 if trainer is from an other company ?

Reply:
dans "from other company c'est pas le trainer du payload le problème c'est la personne qui fait l'appel (le loggedUser), dans le test que j'ai rajouté, la personne qui fait l'appel a le droit de le faire mais ne passe pas le bon trainer en payload